.github/workflows/cpp.yml

```diff
@@ -260,7 +260,7 @@ jobs:
     name: AMD64 Windows MinGW ${{ matrix.mingw-n-bits }} C++
     runs-on: windows-2019
     if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
-    timeout-minutes: 60
+    timeout-minutes: 90
```

Comment:
Seeing that it also times out with 90: https://github.com/apache/arrow/runs/6912383497?check_suite_focus=true
Should we bump it up some more? A long build is annoying but a canceled build is just wasted time...

Reply:
If it times out with 90 then something is going really wrong.